server/eclipse-project/src/main/java/us/freeandfair/corla/csv/DominionCVRExportParser.java

```diff
@@ -306,8 +306,12 @@ private void addContests(final CSVRecord the_choice_line,
       // now that we have all the choices, we can create a Contest object for 
       // this contest (note the empty contest description at the moment, below, 
       // as that's not in the CVR files and may not actually be used)
+      // note that we're using the "Vote For" number as the number of winners
+      // allowed as well, because the Dominion format doesn't give us that
+      // separately
```

Comment:
We know this is true for Colorado, but not for e.g. Pennsylvania, since there are voting methods in use in Pennsylvania that we don't handle. And perhaps note that this won't work (none of it will!) with IRV etc.

Reply:
And yes, this has been discussed with CDOS....

Reply:
It’s not lack of documentation, it’s lack of information - the dominion csv format simply doesn’t distinguish between these things.

Reply:
Got it, so we can't ever know get that info from their format alone, so for now we just have to assume _something_.